client/src/utils/tone/index.ts

```diff
@@ -0,0 +1,83 @@
+import store from 'store';
+import { FlashMessageArg } from '../../components/Flash/redux';
+import { Themes } from '../../components/settings/theme';
+
+// TODO: Add more tone types
+export async function playTone(
+  state: FlashMessageArg['message'] | Themes | string
+): Promise<void> {
+  const playSound = !!store.get('fcc-sound');
+  if (playSound) {
+    const tone = await import('tone');
+    if (tone.context.state !== 'running') {
+      void tone.context.resume();
+    }
+    let player;
+    switch (state) {
+      case 'flash.incomplete-steps':
+        player = new tone.Player(
+          'https://campfire-mode.freecodecamp.org/try-again.mp3'
+        ).toDestination();
+        player.autostart = playSound;
+        break;
+      case 'flash.cert-claim-success':
+        player = new tone.Player(
+          'https://campfire-mode.freecodecamp.org/cert.mp3'
+        ).toDestination();
+        player.autostart = playSound;
+        break;
+      case Themes.Night:
+        const nightToDayPlayer = new tone.Player(
+          'https://campfire-mode.freecodecamp.org/day.mp3'
+        ).toDestination();
+        if (!nightToDayPlayer.loaded)
+          await nightToDayPlayer.load(
+            'https://campfire-mode.freecodecamp.org/day.mp3'
+          );
+        nightToDayPlayer.start();
```

Comment:
Why is this different from the rest?  Can't we use the 

```
player = new Stuff;
player.autostart = true
```
pattern here?

If that's true, we can DRY this via

```js
function playSample(src) {
   const player = new tone.Player(
          'https://campfire-mode.freecodecamp.org/donate.mp3'
        ).toDestination();
   player.autostart = true;
}
```

Reply:
This was a hacky approach during the whole theme/sound saga mess, where the player wouldn't instantiate correctly and needed to be reloaded.

Not sure if it's still necessary, since we've long since adjusted the saga logic.

Reply:
I had a quick play locally and it doesn't seem necessary any more.